docs/static/includes/PR-approval-guidance.md

```diff
@@ -0,0 +1,22 @@
+<!-- markdownlint-disable -->
+<table>
```

Comment:
it took me quite a moment to understand this table. Could we change the format so that the grouping /ccategory is always at the outer layer?

e.g.,

<img width="1012" height="175" alt="Image" src="https://github.com/user-attachments/assets/aebc73b0-67df-4099-9932-cb6cacab65e3" />

Reply:
Your point is valid, and I thought a lot about how to make this the most readable. The Number of owners on the left makes sense, but moving the "who approves the PR?" question doesn't, as the cells below talk about who submits the PR. The who approves the PR is the actual 2*2 matrix in the middle - aka, the content. So, on the top, it could say, "who submits the PR?", but then I would still want to keep the "who approves the PR piece?" as otherwise ever single sentence in the 2*2 would need to end as ".... approves the PR" which makes it much heavier on text.

Reply:
Alright, as with the changes I described above, this was getting way too complicate, I reworked the whole thing similar to what I first had but better: 
<img width="1115" height="197" alt="image" src="https://github.com/user-attachments/assets/55ee1845-4e40-4cee-bda1-efa9ccf054c6" />

Reply:
Good call & much better @matebarabas, thank you. I guess it's one of them 'less is more' situations

Reply:
Good call & much better @matebarabas, thank you. I guess it's one of them 'less is more' situations